### DISCUSSION TITLE
Add HVACMode.Humidify to Climate HVAC Modes

### DISCUSSION THREAD

**Discussion by jwillemsen** [home-assistant/architecture#1077] [Entity Models]
Recently found out that the Daikin Ururu Sarara AC models have a separate dry and humidify mode (see https://github.com/jwillemsen/daikin_onecta/issues/170). Currently there is a HVACMode.DRY in Climate with a description `The device is set to dry/humidity mode.`, would propose to make humidify a separate HVAC mode

**Comment by joostlek** [comment]
Please state your case why we need this and what integrations from core could directly use this feature.

**Comment by jwillemsen** [comment]
Daikin Ururu AC devices have a separate humidify HVAC mode in addition to a dry mode, it is something I think more AC brands support. I searched more and found https://github.com/home-assistant/architecture/issues/288, looks this was asked before. I will close this one and add a remark to #288, there are AC devices which can benefit from this

**Reply by frenck** [reply]
> it is something I think more AC brands support. 

Which ones? Which integrations in core would be able to use this today?

As in, we only extend our entity models, if there are multiple use cases for it in our core codebase.

../Frenck

**Reply by jwillemsen** [reply]
I have search more, but at this moment I can only find Daikin as vendor to ship an AC unit with a built in humidifier, can't find more, seems my remark was incorrect.

**Reply by zen2** [reply]
There is three models of Daikin Ururu Sarara AC with humidifier control:
- FTXZ25N
- FTXZ35N
- FTXZ50N

Corresponding  [Datasheet ](https://my.daikin.eu/content/dam/dauk/document-library/data-sheet/ac/R32/Split_SkyAir/FTXZ-N_RXZ-N_Datasheet_1015.pdf) and  [Operational manual](https://my.daikin.eu/content/dam/document-library/operation-manuals/ac/split/ftxz-n/FTXZ-N_3P338603-1G_Operation%20manual_English.pdf)

Note: I will install one in next weeks.
